examples/provisioning/provision.py

```diff
@@ -23,14 +48,20 @@
 
 # Dnsmasq resources
 for node in nodes_list:
-    dnsmasq = vr.create('dnsmasq_{0}'.format(node['mac'].replace(':', '_')), 'resources/dnsmasq', {})[0]
-    node = filter(lambda n: n.name.endswith('node_{0}'.format(node['mac']).replace(':', '_')), node_resources)[0]
+    node = NodeAdapter(node)
+    node_resource = filter(lambda n: n.name.endswith('node_{0}'.format(node.safe_mac)), node_resources)[0]
+
+    run_result = transport_run.run(node_resource, 'ohai')
```

Comment:
if you really need to `transport_run` there, please at least use `bat` trasport, which will select the best and working transport :) (it's just a matter of changing imports)